docs/DEPLOYMENT.md

```diff
@@ -165,3 +165,17 @@ Check if it complains about any missing environment variables or if there's any
 If you can't figure out what's happening, please [open an issue here](https://github.com/briefercloud/briefer/issues).
 
 </details>
+
+<details>
+  <summary>I can access the web application, but sign-in is not working</summary>
+
+If you're running Briefer from another machine - for example an external server, Raspbery Pi etc - make sure you have enabled HTTPS for your current setup while accessing Briefer's application on port 3000.
+
+In case you can't enable HTTPS or just want to try it out, you can run Briefer in an insecure manner by setting the `ALLOW_HTTP` environment variable to `true`:
```

Comment:
```suggestion
In case you can't enable HTTPS or just want to try it out, you can run Briefer over HTTP by setting the `ALLOW_HTTP` environment variable to `true`:
```

I think it's better to just say HTTP here because it might be that people will deploy a load balancer to terminate SSL in front of it and then it's not necessarily insecure. I think HTTP is precise enough and most people will understand that HTTPS > HTTP.

Wdyt?